docs/src/design.rst

```diff
@@ -25,9 +25,10 @@ Handles and requests
 libuv provides users with 2 abstractions to work with, in combination with the event loop:
 handles and requests.
 
-Handles represent long-lived objects capable of performing certain operations while active. Some
-examples: a prepare handle gets its callback called once every loop iteration when active, and
-a TCP server handle get its connection callback called every time there is a new connection.
+Handles represent long-lived objects capable of performing certain operations while active. For
+example, a prepare handle gets its callback called once every loop iteration when active. Another
+example, is a TCP server handle that gets its connection callback called every time there is a
```

Comment:
I'm not sure about this change. Maybe:

`Some examples: a prepare handle gets its callback called once every loop iteration when active; a TCP server handle that gets its connection callback called every time there is a new connection`

Or even:

```
Some examples: 

- A prepare handle gets its callback called once every loop iteration when active.
- A TCP server handle that gets its connection callback called every time there is a new connection.
```